src/native/src/3rdparty/json/ChangeLog.md

```diff
@@ -0,0 +1,224 @@
+# Change Log
```

Comment:
Instead of committing all the 3rdparty files, can git submodules not be used instead? This would remove the clutter that is created by constantly changing dependencies.